targets/unity-v2/source/Shared/Internal/Log.cs

```diff
@@ -1,7 +1,10 @@
+using System;
+
 namespace PlayFab.Internal
 {
     public static class Log
     {
+        [Obsolete("Debug has deprecated.")]
```

Comment:
I think either "Debug has been deprecated." or "Debug is deprecated." would flow a bit easier.